openapi/docs/LocalstatequeryAPI.md

```
@@ -366,3 +367,71 @@ No authorization required
+### Path Parameters
+
+
+
+### Other Parameters
+
+Other parameters are passed through a pointer to a apiLocalstatequeryUtxosSearchByAssetGetRequest struct via the builder pattern
+
+
+Name | Type | Description  | Notes
+------------- | ------------- | ------------- | -------------
+ **policyId** | **string** | Policy ID (hex) | 
```

markdownlint-cli2 (0.22.1)

[warning] Table pipe style
Expected: no_leading_or_trailing; Actual: trailing_only; Unexpected trailing pipe

(MD055, table-pipe-style)